eips_code/eip-3540.py

```diff
@@ -64,11 +63,11 @@ def validate_code(code: bytes) -> bool:
 # Legacy contracts
 assert validate_code(b'') == True
 assert validate_code(b'\x00') == True
-assert validate_code(b'\xef') == True  # Magic byte missing
+assert validate_code(b'\xef') == True  # Magic second byte missing
 
-# Any value outside the magic
-for magic in range(1, 256):
-    assert validate_code(bytes((0xEF, magic))) == True
+# Any value outside the magic second byte
```

Comment:
This comment may make me think this change is not a good idea after all 😅

Reply:
Something that made me think that was: implementations will still have to check for `0xEF` separately, in the part `else if *code* starts with `0xEF`, creation continues to result in an exceptional abort (the rule introduced in EIP-3541)`

Reply:
This is mostly issue how to define and name constants: `0xEF` and `0x00` vs `0xEF` and `0xEF00`.